examples/sample-app/application-template-custombuild.json

```diff
@@ -145,6 +145,9 @@
             "name": "origin-ruby-sample:latest"
           }
         },
+        "postCommit": {
+          "script": "bundle exec rake test"
+        },
```

Comment:
Perhaps we should omit this example, as the hook is not run by default for custom builds, @bparees wdyt?